core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -1908,6 +1909,180 @@ boolean scheduleTask(WorkflowModel workflow, List<TaskModel> tasks) {
         return startedSystemTasks;
     }
 
+    List<TaskModel> getActualTasksToBeQueued(
```

Comment:
can we name this like `getTasksToBeQueued`, calling `actual` seems like there is also some `fake` task that can be scheduled.